src/main/java/nextstep/subway/line/domain/entity/LineSections.java

```diff
@@ -173,6 +166,48 @@ private boolean isContainUpStations(Station compareStation) {
             .anyMatch(station -> station.equals(compareStation));
     }
 
+    private void validateDeletelineSection() {
+        if(lineSections.size() <= 1) {
```

Comment:
전체적으로 코드 포맷팅 한 번 확인해주시면 좋을 것 같아요.
```suggestion
        if (lineSections.size() <= 1) {
```